lib/linguist/languages.yml

```diff
@@ -6581,6 +6581,14 @@ Windows Registry Entries:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 969674868
+Witcher Script:
+  type: programming
+  color: "#ff0000"
```

Comment:
How did you decide upon this colour? Peeps are likely to ask in future so it would be nice to know.

Reply:
Completely subject to change but the company that made the Witcher is CD PROJEKT **RED**.

Reply:
I could also use #db2813 if needed?

Reply:
Whichever you prefer.

Reply:
I think I'll keep it there for now, if needed I can likely change it later.